.travis.yml

```diff
@@ -1,26 +1,19 @@
-matrix:
-  include:
-  - name: "Node.js: 12"
-    language: node_js
-    cache: yarn
-    node_js:
-    - '12' # current stable
-    script: yarn test --runInBand
-  - name: "Ruby: 2.5.8"
-    language: ruby
-    cache: bundler
-    rvm:
-    - 2.5.8
-    addons:
-      postgresql: '10'
-    install: bin/setup
-    after_script: bin/ci/after_script
-  - name: "Ruby: 2.6.6"
-    language: ruby
-    rvm:
-    - 2.6.6
-    cache: bundler
-    addons:
-      postgresql: '10'
-    install: bin/setup
-    after_script: bin/ci/after_script
+---
+language: ruby
+node_js: 12
+cache:
+  bundler: true
+  yarn: true
+addons:
+  postgresql: '10'
+rvm:
+- 2.5.8
+- 2.6.6
```

Comment:
Super minor, but I think we should keep the languages/versions together, so can this be moved up between the language and node_js key?

Reply:
:+1: done